tools/build/CreateRelease.php

```diff
@@ -88,18 +88,6 @@
     exit(0);
 }
 
-foreach ($releaseOptions as $optionName => $option) {
```

Comment:
Hello, so PHPStan says this is dead code? `$option['required']` and `$userOptions[$optionName]` cannot be set right?

Reply:
Affirmative. Why ? Because `$required` will be always defined to `false` so it never go in `if` section.